api/src/pcapi/routes/backoffice_v3/individual_bookings.py

```diff
@@ -245,3 +246,77 @@ def mark_booking_as_cancelled(booking_id: int) -> utils.BackofficeResponse:
+
+
+@individual_bookings_blueprint.route("/batch-validate", methods=["GET"])
+def get_batch_validate_indivual_bookings_form() -> utils.BackofficeResponse:
+    form = booking_forms.BatchActionOnBooking()
+    return render_template(
+        "components/turbo/modal_form.html",
+        form=form,
+        dst=url_for("backoffice_v3_web.individual_bookings.batch_validate_individual_bookings"),
+        div_id="batch-validate-booking-modal",
+        title="Voulez-vous vraiment valider les réservations ?",
+        button_text="Valider les réservations",
+    )
+
+
+@individual_bookings_blueprint.route("/batch-validate", methods=["POST"])
+def batch_validate_individual_bookings() -> utils.BackofficeResponse:
+    return _batch_individual_bookings_action(
+        success_message="Les réservation ont été validées avec succès", validate=True
+    )
+
+
+@individual_bookings_blueprint.route("/batch-reject", methods=["GET"])
+def get_batch_reject_indivual_bookings_form() -> utils.BackofficeResponse:
+    form = booking_forms.BatchActionOnBooking()
+    return render_template(
+        "components/turbo/modal_form.html",
+        form=form,
+        dst=url_for("backoffice_v3_web.individual_bookings.batch_reject_individual_bookings"),
+        div_id="batch-reject-booking-modal",
+        title="Annuler les réservations",
+        button_text="Annuler les réservations",
+    )
+
+
+@individual_bookings_blueprint.route("/batch-reject", methods=["POST"])
+def batch_reject_individual_bookings() -> utils.BackofficeResponse:
+    return _batch_individual_bookings_action(
+        success_message="Les réservation ont été annulées avec succès", validate=False
+    )
+
+
+def _batch_individual_bookings_action(success_message: str, validate: bool = False) -> utils.BackofficeResponse:
+    form = booking_forms.BatchActionOnBooking()
+    try:
+        booking_ids = [int(id) for id in form.object_ids.data.split(",")]
+    except ValueError:
+        flash("Identifiant(s) invalide(s)")
+        return _redirect_after_individual_booking_action(400)
+    except AttributeError:
+        flash("Erreur sur les indentifiants")
+        return _redirect_after_individual_booking_action(403)
+
+    bookings = bookings_models.Booking.query.filter(bookings_models.Booking.id.in_(booking_ids)).all()
+    api_function = bookings_api.mark_as_cancelled
+    if validate == True:
+        api_function = bookings_api.mark_as_used
+
+    for booking in bookings:
+        try:
+            api_function(booking)
+        except bookings_exceptions.BookingIsAlreadyCancelled:
+            if validate == True:
+                bookings_api.mark_as_used_with_uncancelling(booking)
+            else:
+                flash("Une des réservation a déjà été annulée", "error")
+                return _redirect_after_individual_booking_action(400)
+        except bookings_exceptions.BookingIsAlreadyUsed:
+            flash("Une des réservation a déjà été validée", "error")
```

Comment:
```suggestion
            flash("Au moins une des réservations a déjà été validée", "error")
```
ou poursuivre et afficher le nombre d'erreurs